### ISSUE TITLE
Update Stylelint

### ISSUE THREAD

**Issue by vbalcom** [SapientNitroLA/sugarcoat#35]
```
Deprecation Warning: 'block-no-single-line' has been deprecated and in 8.0 will be removed. Instead use 'block-opening-brace-newline-after' and 'block-closing-brace-newline-before' with the "always" option. See: https://stylelint.io/user-guide/rules/block-no-single-line/
Deprecation Warning: 'stylelint-disable-reason' has been deprecated and in 8.0 will be removed. See: https://stylelint.io/user-guide/rules/stylelint-disable-reason/
```

**Comment by vbalcom** [comment]
#37

**Comment by tisAliG** [comment]
Merged #37, closing.
